docs-ref-conceptual/azure-cli-manage-secrets.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Manage Azure secrets using the Azure CLI | Microsoft Docs
+description: Learn to manage Azure passwords and secrets generated by Azure CLI commands.
+author: dbradish
+ms.author: dbradish-microsoft
+ms.date: 01/31/2024
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+---
+
+# Use Azure CLI to manage sensitive Azure information such as keys, passwords and secrets
+
+When you manage Azure resources, there is sensitive information that must be protected. For example, keys, passwords and connection strings can be created by Azure CLI commands and displayed in a terminal window.
+It is critical to protect this information which, when acquired publically, can cause serious damage and lead to a loss of trust in your company's products and services.
+
+## Set show_secrets_warning (phase 1)
+
+Beginning in Azure CLI x.y.z, a warning message will display when reference commands result in the output of sensitive information. The purpose of the warning message is to decrease the unintentional exposure of secrets, but these messages may require you to make changes in existing scripts.
+
+> [!IMPORTANT]
+> The new warning messages are sent to Standard Error (STDERR), not Standard Out (STDOUT).
+> Therefore, if you are running an Azure CLI command that results in sensitive information output, you will need to trap for the warning message or turn warnings off using `az config set show_warnings = off`.
+
+The Azure CLI identifies "sensitive information" by **{infomation from engineering here}.**
+
+Turn sensitive information warnings `on` and `off` by setting the `show_secrets_warning` configuration property.
+
+```azurecli-interactive
+az config set show_warnings = off
+```
```

Comment:
```suggestion
```azurecli-interactive
az config set clients.show_secrets_warning=True
```
```

Reply:
@dcaro, in [azure-cli-configuration.md](https://learn.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest) we use `yes` and `no`, so I changed `off` to `yes`.